manifests/f/flxzt/rnote/0.7.1/flxzt.rnote.locale.en-US.yaml

```
@@ -0,0 +1,43 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: flxzt.rnote
+PackageVersion: 0.7.1
+PackageLocale: en-US
+Publisher: flxzt
```

```suggestion
Publisher: Felix Zwettler
```
Must match Control Panel / Apps&Features for upgrade matching from WinGet's side